python/tutorial1_display.py

```diff
@@ -79,10 +52,7 @@
     outputs.append(Buffer(Type(TypeCode.Uint, depth_of_buffer, 1), output_size))
 
     # set I/O ports
-    port_map = PortMap()
-    port_map.set_buffer(gain_p, gains)
-    port_map.set_buffer(exposuretime_p, exposures)
-    port_map.set_buffer_array(output_p, outputs)
+    output_p[0].bind(outputs[0])
```

Comment:
after pull https://github.com/fixstars/ion-kit/pull/207
you can use `output_p.bind(outputs)`